src/views/Auctions/components/AuctionClock/index.tsx

```diff
@@ -11,20 +11,39 @@ import { Auction } from 'src/interfaces/Auction'
 import { CardText } from 'src/components/CardText'
 import { isAuctionClosed, isAuctionUpcoming } from 'src/mesa/auction'
 
+
+// Utils
+import { convertUtcTimestampToLocal } from 'src/utils/date'
+
+
 interface AuctionClockProps {
   auction: Auction
 }
 
+
+export const timerPercentage = (auction: Auction)  => {
+  const localTimeStamp = dayjs(Date.now()).unix()
+  const startTime = convertUtcTimestampToLocal(auction.startBlock)
+  const endTime = convertUtcTimestampToLocal(auction.endBlock)
+  const totalTime = Math.abs(startTime - endTime)
+  const percentage = (1 - Math.abs(localTimeStamp - endTime) / totalTime) * 100
+  return percentage
+}
+
+
+
 export const AuctionClock: React.FC<AuctionClockProps> = ({ auction }) => {
+  
+  const color = '#304ffe';
+
+
+
   if (isAuctionClosed(auction)) {
     return (
       <Flex flexDirection="row" justifyContent="space-between">
         <CardText color="grey">Closed</CardText>
         <Flex>
           <Timer auction={auction} />
-          <DarkCircle>
-            <LightCircle />
-          </DarkCircle>
         </Flex>
       </Flex>
     )
```

Comment:
Do we need "else if"?
I think "if" is enough

Reply:
I wasn't able to simplify this further. I think I do need the `else if` as there are 3 states the auction could be in; open, closed and upcoming. I have made the open auction the default state and used conditionals to choose between the other two states.

If there is another way, could you give me a quick example?

Reply:
Once you return something, then it will end up running that function.
So no need to use "else if" if the previous "if" returns something.
You can use 2 "if" statements without "else if"